src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Ssl.cs

```diff
@@ -279,4 +290,86 @@ private SafeSslHandle() : base(IntPtr.Zero, true)
         {
         }
     }
+
+    internal sealed class SafeChannelBinding : SafeHandle
+    {
+        [StructLayout(LayoutKind.Sequential)]
+        private struct SEC_CHANNEL_BINDINGS
+        {
+            internal int dwInitiatorAddrType;
```

Comment:
Do we actually need a structure that is similar to the windows structure?  From [MSDN](https://msdn.microsoft.com/en-us/library/vstudio/system.security.authentication.extendedprotection.channelbinding(v=vs.110).aspx)
>The ChannelBinding class encapsulates a pointer to the opaque data used to bind an authenticated transaction to a secure channel.

Since it is "opaque data", that leads me to believe we can use any structure we want here.

Am I wrong? Do callers actually inspect this data?